frontend/src/tests/nonprofitApply/nonprofitApplyTitle.test.js

```diff
@@ -0,0 +1,10 @@
+import React from 'react'
```

Comment:
(nit) can change name to criteriaTitle.test.js

Reply:
done